contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+     [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for a
+     self-contained virtual machine running kubernetes in VirtualBox - - See
+     [minikube specific notes](#minikube).
+ * Local development machine dependencies:
+   * [GNU Bash](https://www.gnu.org/software/bash/) and friends. If you're on
+     Linux or Mac, you should be good to go.
+   * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+      * Tested with kubectl v1.14.0
+   * [helm](https://github.com/helm/helm/releases)
+      * Tested with helm v2.13.1
+ * Optional: for TLS / HTTPs support, you will also need an internet domain
+   name, and the ability to update its DNS.
+
+Your cloud provider should have instructions for setting up `kubectl` to talk to
+your cluster. This usually involves downloading a config file and putting it in
+`$HOME/.kube/config`. (The file has to be renamed `config` and put in the
+`$HOME/.kube` directory.)
+
+Test that your `kubectl` can talk to your cluster, by querying for a list of
+running nodes:
+
+```
+kubectl get nodes
+```
+
+If everything is working, you should see a list of one or more nodes running and
+showing `STATUS=Ready`
+
+## Security Notice
+
+Any cryptocurrency wallet that is online is a security concern. For any
+real-world production deployment, you will need to review this architecture
+closely to see if it fits with your chosen platform and network environment.
+
+This system is currently designed for a kubernetes cluster that has a single
+administrator (or possibly a small team of trusted users). It will not support
+untrusted multi-tenancy out of the box.
+
+All of the services are created in their own namespace, but no [Security
+Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) have
+been applied to the pods.
+
+The Helm configuration file contains *all* of the configuration for the system,
+*including passwords* in plain text.
+
+The lbrynet SDK wallets are individually stored unencrypted in their own
+persistent volumes.
+
+## Installation
+
+This system is installed via [Helm](https://helm.sh/docs/), the package manager
+for Kubernetes. [Helm Charts](https://helm.sh/docs/developing_charts/#charts)
+are the basis for packages in Helm. This directory is a Helm chart itself.
+
+All of the helm and kubectl commands necessary to install, upgrade, and maintain
+your deployments, are wrapped in the included [`run.sh`](run.sh) script. For
+debugging purposes, this wrapper also prints to stdout the full underlying
+commands (helm, kubectl, etc) as they are run.
+
+### Create a project directory
+
+Create a new directory someplace to store your deployment configuration. For the
+rest of this tutorial, you will work from this directory:
+
+```
+mkdir $HOME/k8s-lbry-test
+
+cd $HOME/k8s-lbry-test
+```
+
+Download `run.sh` to this same directory:
+
+```
+curl -Lo run.sh https://raw.githubusercontent.com/EnigmaCurry/lbry-docker/k8s-lbry/contrib/k8s-lbry/run.sh
+
```

Comment:
Possibly put a comment here mentioning that people should always review the contents of a curl'd script for secure operations involving money.

Reply:
added